selectrum.el

```diff
@@ -1277,6 +1265,9 @@ CANDIDATES is the list of strings that was passed to
 list and sorted first. If `minibuffer-default' is set it will
 have precedence over DEFAULT-CANDIDATE."
   (setq-local selectrum-active-p t)
+  (unless selectrum-refine-candidates-function
+    (setq-local selectrum-refine-candidates-function
```

Comment:
Why? Why not set it as default?

Reply:
I was going to do that, as I said this is not ready ;)

Reply:
Then it should probably also be made public again?

Reply:
You can do either way, but public might be more reasonable, I think.

Reply:
I think its okay to references private functions with public variables but I suspect there was a reason the default functions where public before...